nylas/client/client.py

```diff
@@ -494,3 +499,21 @@ def _call_resource_method(self, cls, id, method_name, data):
 
         result = _validate(response).json()
         return cls.create(self, **result)
+
+    def _request_neural_resource(self, cls, data, path=None, method="PUT"):
+        if path is None:
+            path = cls.collection_name
+        url = URLObject(self.api_server).with_path("/neural/{name}".format(name=path))
+
+        session = self._get_http_session(cls.api_root)
+
+        converted_data = convert_datetimes_to_timestamps(data, cls.datetime_attrs)
+        response = session.request(method, url, json=converted_data)
+
+        result = _validate(response).json()
+        # The Neural API will always return only one item
+        # but sometimes it returns it in the form of a 'singleton' array
```

Comment:
Is this the restriction of the python SDK? `conversation` and `signature` can actually return list of objects.

Reply:
If the endpoints get more than 1 message id, it will return multiple messages

Reply:
Oops, I did not know that. I thought from the documentation it was always 1:1. I will update the code to reflect this.

Reply:
@egginsect I have pushed a change to reflect your comments. Does the same apply to OCR? Or does OCR only support one file ID at a time? Thanks!

Reply:
Yeah it takes only 1 file_id per request